packages/io/src/io.ts

```diff
@@ -278,7 +280,11 @@ export async function findInPath(tool: string): Promise<string[]> {
 function readCopyOptions(options: CopyOptions): Required<CopyOptions> {
   const force = options.force == null ? true : options.force
   const recursive = Boolean(options.recursive)
-  return {force, recursive}
+  const copySourceDirectory =
```

Comment:
why not do what force does? `const force = options.force == null ? true : options.force`
It also defaults to true.

Reply:
I think `undefined` is actually the value we should check against since `== null` is doing type coercion and they just happen to be equal based on the implementation of `==` I'll update both to use `===`